docker/pythonpath/superset_config.py

```diff
@@ -30,6 +30,7 @@
 from flask_appbuilder.security.views import AuthOAuthView
 from flask_appbuilder import expose
 from flask import flash, get_flashed_messages
+from flask_babel import gettext as _, get_locale
```

Comment:
```suggestion
from flask_babel import get_locale
```

Is `gettext as _` needed?